code/__defines/skills.dm

```diff
@@ -12,25 +12,14 @@
 #define SKILL_AVERAGE  2
 #define SKILL_HARD     4
 
-#define SKILL_BUREAUCRACY   /decl/hierarchy/skill/organizational/bureaucracy
-#define SKILL_FINANCE       /decl/hierarchy/skill/organizational/finance
 #define SKILL_EVA           /decl/hierarchy/skill/general/EVA
-#define SKILL_MECH          /decl/hierarchy/skill/general/mech
-#define SKILL_PILOT         /decl/hierarchy/skill/general/pilot
```

Comment:
Any space setting has ships, i don't think removing piloting is a good move

Reply:
Piloting is still in, looking at one of my earlier commits - removed it by mistake before rectifying it in a later commit.